docs-nextra/content/infrastructure/maskfile.mdx

```diff
@@ -0,0 +1,356 @@
+)
+```
+
+### ecs build:all
+
+> Use `act` CLI to build and push all containers with local code, tagged with the HEAD (or HEAD-dirty) SHA
+
+No options are required -- the workflow infers them all.
+
+**WARN**: `docker push` invocations will appear to freeze, but that is a display bug in `act`.
+
+```bash
+
+( cd ..
+    workflow_file=".github/workflows/ecrbuild-all.yml"
+
+    echo "This may take a few minutes, and output will not stream during upload ..."
+    echo "Procedure will follow workflow $workflow_file ..."
+    act \
+      -W "$workflow_file" \
+      --container-architecture linux/amd64 \
+      workflow_call
+
+    echo "Done!"
+)
+```
+
+### ecs bastion
+
+> Opens an interactive shell on the bastion container in AWS
+
+**OPTIONS**
+
+- region
+    - flags: -r --region
+    - type: string
+    - desc: Which AWS region to use (default us-east-1)
+- proper_name
+    - flags: -n --proper-name
+    - type: string
+    - desc: proper name of AWS environment (see `./aws` module); e.g. blake
+    - required
+- environment
+    - flags: -e --environment
+    - type: string
+    - desc: environment name of AWS environment (see `./aws` module) e.g. staging
+    - required
+
+```bash
+AWS_REGION=${region:-us-east-1}
+
+echo "fetching task ID of running bastion ..."
+TASK=$(
+    aws ecs \
+      list-tasks \
+      --region ${AWS_REGION} \
+      --cluster ${proper_name}-ecs-cluster-${environment} \
+      --service ${proper_name}-bastion \
+        | jq -r \
+            '.taskArns[0]' \
+        | cut \
+            -d'/' \
+            -f 3 \
+)
+
+echo "starting shell with task $TASK ..."
+aws ecs \
+  execute-command \
+  --interactive \
+  --command "/bin/sh" \
+  --region ${AWS_REGION} \
+  --container "bastion" \
+  --cluster ${proper_name}-ecs-cluster-${environment} \
+  --task $TASK
+```
+
+{/* build nginx container */}
+
+### ecs nginx:build
+
+> Builds the nginx container used in AWS ECS for inbound traffic
+
+```bash
+echo "building Nginx container..."
+docker build \
+  --platform linux/amd64 \
+  -t pubpub-v7-nginx:latest \
+  ./nginx
+```
+
+### ecs nginx:push
+
+> Pushes the locally built latest nginx container
+
+**OPTIONS**
+
+- region
+    - flags: -r --region
+    - type: string
+    - desc: Which AWS region to use (default us-east-1)
+
+```bash
+echo "Determining AWS Account ID..."
+
+AWS_REGION=${region:-us-east-1}
+AWS_ACCOUNT=$(
+  aws sts get-caller-identity \
+    --query Account \
+    --output text
+)
+AWS_REGISTRY=$AWS_ACCOUNT.dkr.ecr.$AWS_REGION.amazonaws.com
+
+echo "Logging docker daemon in to ECR"
+aws ecr get-login-password \
+  --region $AWS_REGION \
+  | docker login \
+      --username AWS \
+      --password-stdin \
+      $AWS_REGISTRY
+
+echo "renaming container to ECR repository..."
+docker tag \
+  pubpub-v7-nginx:latest \
+  $AWS_REGISTRY/nginx:latest
+
+echo "pushing Nginx container..."
+docker push \
+  $AWS_REGISTRY/nginx:latest
+```
```

Comment:
Thinking we should remove the maskfile from the docs since I'm not sure if it'd execute properly from this directory + it could get out of sync with the maskfile in our infrastructure directory.